plugins/sbt/graphql-java-codegen-sbt-plugin/version.sbt

```diff
@@ -0,0 +1 @@
+version in ThisBuild := "0.0.2-SNAPSHOT"
```

Comment:
Please align the version of the plugin with the version of all other plugins.

Reply:
please see pr's file list,